src/parser.cu

```diff
@@ -418,6 +429,10 @@ int checkParameterValidity(
     dout(V_ERROR) << "At least 1 repetition is necessary!" << std::endl;
   }
 
+  if(adaptiveSteps < 1){
```

Comment:
The default value is 5, to enable adaptive behavior like before (the macro said 5).
This `if`-statement only triggers, if the user **explicitly** sets the value to something smaller than 1.